healthcheck/TemplateHealthcheck.html

```diff
@@ -0,0 +1,27 @@
+  <h2>{{metadata.name}} - Running against database "{{metadata.db_name}}"</h2>
```

Comment:
This is nitpicky, but I prefer to have spaces between the curly braces and content, like so: {{ metadata.name }}.  This is a bit more readable, what do you think?